src/Adapter/Currency/CommandHandler/ToggleExchangeRateAutomatizationHandler.php

```diff
@@ -180,10 +180,13 @@ private function getCronUrl()
      */
     private function createCronJob($cronUrl)
     {
+        if (!class_exists(CronJobs::class)) {
+            return false;
+        }
         /** @var CronJobs $cronJobsModule */
         $cronJobsModule = Module::getInstanceByName('cronjobs');
```

Comment:
```suggestion
        /** @var CronJobs $cronJobsModule */
        $cronJobsModule = Module::getInstanceByName('cronjobs');
        if (empty($cronJobsModule)) {
            return false;
        }
```
Just test if the module is correctly loaded.

Reply:
Fixed